### PR TITLE
Added environment variables for languange-mogic-testrunner

### PR DESCRIPTION
**Pull request by hguhlich**

### CODE REVIEW

File: templates/language-mogic-testrunner/0/rancher-compose.yml [hunk 1]
```diff
@@ -28,10 +28,34 @@ version: '2'
         - webui_external
       required: true
 
-#lb:
-#  scale: 2
-#  load_balancer_config:
-#    name: lb config
+    - variable: db_host
```
Comment (flypenguin) [inline templates/language-mogic-testrunner/0/rancher-compose.yml]: I think the config tool uses only two variables, which *I* personally think is nicer. it looks like this: `user@db_host:1234/db_name`, and is split in the starting container. maybe you want to use this as well? just as a comment :)

File: templates/language-mogic-testrunner/0/docker-compose.yml [hunk 2]
```diff
@@ -32,7 +37,7 @@ services:
     image: hub.cognotekt.int:5000/language/mogic/parser
     ports:
       - "8080"
-    mem_reservation: 536870912
+    mem_reservation: 134217728
```
Comment (flypenguin) [inline templates/language-mogic-testrunner/0/docker-compose.yml]: :)

File: templates/language-mogic-testrunner/0/docker-compose.yml [hunk 3]
```diff
@@ -7,6 +7,11 @@ services:
     environment:
       PREPROC_HOST: http://preprocessor:8081
       PARSER_HOST: http://parser:8080
+      DB_HOST: ${db_host}
```
Comment (flypenguin) [inline templates/language-mogic-testrunner/0/docker-compose.yml]: see comment below.

File: templates/language-mogic-testrunner/0/rancher-compose.yml (hunk 1 above)
Comment (kebleynad) [inline templates/language-mogic-testrunner/0/rancher-compose.yml]: Definitely, but later.

**Pull request merged**